adcm_aio_client/core/objects/cm.py

```diff
@@ -225,10 +237,53 @@ def cluster(self: Self) -> Cluster:
     def components(self: Self) -> "ComponentsNode":
         return ComponentsNode(parent=self, path=(*self.get_own_path(), "components"), requester=self._requester)
 
+    @property
+    def license(self: Self) -> License:
+        return License(self._requester, self._data)
+
+    @async_cached_property
+    async def maintenance_mode(self: Self) -> MaintenanceMode:
+        response = await self._requester.get(*self.get_own_path())
+        self._data = response.as_dict()
```

Comment:
I see no reason for re-requesting object for maintenance mode sake

see comment in your's another MR on this matter